import-map.test.json

```diff
@@ -0,0 +1,22 @@
+{
+  "imports": {
+    "hooks/usePrefix.ts": "./tests/mocks/mockUsePrefix.ts",
+    "hooks/usePrint.ts": "./tests/mocks/mockUsePrint.ts",
+    "path": "./src/vendor/Path.ts",
+    "types": "./src/types.ts",
+    "hooks": "./src/hooks/index.ts",
+    "hooks/": "./src/hooks/",
+    "prefab": "./src/prefab/index.ts",
+    "prefab/": "./src/prefab/",
+    "deno/": "https://deno.land/std@0.176.0/",
```

Comment:
I wish there was a way to extend/overlay import maps, so we didn't have to maintain a dep in two places. I particular, I've run into things where I need a remote import map + a local map, or two remote maps. This would be very nice to have in deno.
```json
"extends": "./src/import-map.json",
"hooks/": "./tests/mocks/hooks/",
```

Reply:
When I finish the refactor I think it probably won't need this at all anyways.  Deno docs recommend using an `_internal` function table for doing mocks and once we've consolidated our a lot of the deno calls into the hooks mocking will be much easier.